src/resources/views/crud/inc/datatables_logic.blade.php

```diff
@@ -209,9 +209,9 @@ functionsToRunOnDataTablesDrawEvent: [],
               "type": "POST"
           },
           dom:
-            "<'row hidden'<'col-sm-6 hidden-xs'i><'col-sm-6 hidden-print'f>>" +
+            "<'row hidden'<'col-sm-6'i><'col-sm-6 d-print-none'f>>" +
```

Comment:
Just `print` fixes - looks like we didn't change the classes when moving from Bootstrap 3 to Bootstrap 4.